libs/@local/hash-authorization/src/backend/spicedb/api.rs

```diff
@@ -66,6 +66,9 @@ impl fmt::Display for StreamError {
 
 impl Error for StreamError {}
 
+type StreamReturn<T: DeserializeOwned, B: Serialize + Sync> =
```

Comment:
too bad that TAITs are currently the only solution to overcapturing :/

Reply:
Yup, agreed. Do you think we should add a todo in the case this can be changed? Or should we just wait and revisit the code base at that point (Use a regex for type aliases and see if it's an `impl trait` type alias?) As we added the feature in this PR it would be easy to add a comment everywhere but it's also likely that this will be missed in future work and will be forgotten in the future. I would probably just leave it without comment.

Reply:
I agree, don't think a comment here is really helpful considering that we're going to use TAITs sooner or later in more places. Once a more concise syntax is finally proposed and implemented we can grep to implement as that will likely find more than just these initial comments.